mv: if more than one of -i, -f, -n is specified, only the final one takes effect (#4826)
uutils/coreutils: src/uu/mv/mv.md

## After Help

> If you specify more than one of `-i`, `-f`, `-n`, only the final one takes effect.

Reviewer: ```suggestion
When specifying more than one of -i, -f, -n, only the final one will take effect.
```
